### DISCUSSION TITLE
Widget title

### DISCUSSION THREAD

**Discussion by Schnok** [templaza/astroid-framework#770] [Q&A]
Hello everyone,

I've just started using Astroid and I'm trying to display the widget title. I thought that this option was probably right in front of me, but I can't find it :/. Is this possible?
Thank you for your help.

Best regards,
Phil

**Comment by sonvnn** [comment]
Widget title is a way to identify in layout builder only. If you want to display a Heading tag before a Widget. Please use Heading Widget.

**Comment by Schnok** [comment]
Thank you @sonvnn . I knew it would be easy...
